msteams-platform/sbs-inmeeting-document-signing.yml

```diff
@@ -0,0 +1,473 @@
+    
+    1. Select **Add a client application** in the **Authorized client applications** section. 
+
+        :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/add-client-application.png" alt-text="Select client application":::
+
+       The **Add a client application** section appears.
+
+    1. Enter the client application details:
+
+         :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/authorizedscope1.png" alt-text="Add client application 2":::
+
+        1. Enter the **Client ID** for for the application that you want to authorize. It's the application ID of the client that you want to authorize. Use the following IDs for client applications that you want to authorize:
+
+           | **Client ID** | **For authorizing** |
+           | --- | --- |
+           | `1fec8e78-bce4-4aaf-ab1b-5451cc387264` | Teams desktop or mobile application |
+           | `5e3ce6c0-2b1f-4285-8d4b-75ee78787346` | Teams web application |
+         
+           > [!NOTE]
+           > You'll need to add authorization for desktop and web application separately.
+
+        1. Select **Authorized scopes** to determine which type of token is returned by the service.
+
+        The following image displays the client ID:
+
+        :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/clientapps.png" alt-text="Client applications"::: 
+  
+    **Configure Graph permissions**
+
+    1. In the left panel, select **API Permissions**. 
+
+        :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/select-api-permissions.png" alt-text="Select API permissions"::: 
+        
+       > [!NOTE]
+       > Users need to consent to these permissions only if the Azure AD app is registered in a different tenant.
+
+    1. Select **Add a permission**.
+
+        :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/addpermission.png" alt-text="Add permission"::: 
+
+    1. Select **Microsoft Graph**.
+
+        :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/microsoft-graph.png" alt-text="Microsoft Graph"::: 
+
+    1. Select **Delegated permissions**.
+
+        :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/delegated-permission.png" alt-text="Delegated-permission"::: 
+
+    1. Select the Graph permissions:
+
+        :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/permissions.png" alt-text="Delegated-permission":::
+
+        1. From the list of permissions, select the following:</br>
+         * **email**
+         * **offline_access**
+         * **OpenId**
+         * **profile**
+         * **User.Read**
+         * **User.ReadBasic.All**
+
+        1. Select **Add permissions**.  
+
+    **To set redirect URI for configuring platform**
+
+    1. From the left panel, select **Authentication** to set a redirect URI. 
+
+       > [!NOTE]
```

Comment:
I'm not sure this is correct, from my recollection I don't have any admin specific permissions in the app, and I don't see it in the Azure portal today. Is this something you saw during testing?

Reply:
Either way, I'm not sure it makes sense to include it in the redirect uri section, maybe it's better in the "Configure Graph permissions" section

Reply:
@eoinobrien please let me know if you're referring to the path for 'Authentication' option, or the section for setting up the redirect URI. Should I remove the section?

Reply:
The note about the consent is from the SSO for tabs section.